src/tikzdocument.jl

```diff
@@ -366,6 +366,6 @@ function Base.display(::PGFPlotsXDisplay, p::_SHOWABLE)
             error("Failed to show the generated pdf, run `PGFPlotsX.enable_interactive(false)` to stop trying to show pdfs.\n", "Error: ", sprint(Base.showerror, e))
         end
     else
-        print(io, p)
+        throw(MethodError(display, (d, p)))
```

Comment:
I think the original intention here was just to print the tex to the REPL. It seems a bit odd to throw a method error depending on a setting.

Reply:
Throwing a `MethodError` is the documented way of signaling that `display` can't display the object so the outer `display` just walks down the list of displays. See [`Base.Multimedia.display`](https://docs.julialang.org/en/v1/base/io-network/#Base.Multimedia.display).

Displaying the LaTeX representation is possible with a `display(print_tex(String, p))` here, but I am not sure why we would do that. We have `print_tex` for this.